spec/controllers/characters_controller_spec.rb

```diff
@@ -74,9 +74,11 @@
       it "skips retired characters when specified" do
```

Comment:
probably want to rename this and also test that it hides npcs even when retired status isn't specified? might be better to just make a test for that tbh

Reply:
Not sure why I combined this, it really should've been separate - thanks!